src/runtime_src/core/pcie/common/system_pcie.cpp

```diff
@@ -35,11 +35,14 @@ get_devices(boost::property_tree::ptree& pt) const
     // Key: device_id
     pt_device.put("device_id", std::to_string(device_id));
 
-    // Key: pcie
+    // Key: platform_name
```

Comment:
I was expecting to see a system_edge.cpp like file for the Edge devices to populate its unique metadata.  

I was not expecting it to re-use the PCIe code.  What insights do you have that I am missing?